spk/cops/src/wizard_templates/install_uifile.yml

```
@@ -0,0 +1,13 @@
+COPS_CONFIGURATION_FIRST_STEP_TITLE: "COPS configuration"
+
+EXISTING_CALIBRE_DIRECTORY_DESCRIPTION: "Existing Calibre library directory. WARNING: has to end with a / and never use your web directory"
+EXISTING_CALIBRE_DIRECTORY_LABEL: "Calibre directory"
+COPS_CATALOG_TITLE_DESCRIPTION: "Catalog title."
+COPS_CATALOG_TITLE_LABEL: "Catalog title"
+
+COPS_CONFIGURATION_SECOND_STEP_TITLE: "COPS configuration (next)"
+DO_YOU_WANT_TO_USE_COPS_WITH_A_KOBO_DESCRIPTION: "Do you want to use COPS with a Kobo eReader?"
+DO_YOU_WANT_TO_USE_COPS_WITH_A_KOBO_LABEL: "Kobo"
+
+DSM_PERMISSIONS_TITLE: "Attention! DSM Permissions"
+DSM_PERMISSIONS_TEXT: "Package user and group will not appear on most UI settings. Please read <a target=\"_blank\" href=\"https://github.com/SynoCommunity/spksrc/wiki/Permission-Management\">Permission Management</a> for details."
```

@smaarn I you wonder, why you get invalid packages with the latest change, you have to qote the `\` and the `"` like this

`target=\\\"_blank\\\"`

to generate a valid json file

```suggestion
DSM_PERMISSIONS_TEXT: "Package user and group will not appear on most UI settings. Please read <a target=\\\"_blank\\\" href=\\\"https://github.com/SynoCommunity/spksrc/wiki/Permission-Management\\\">Permission Management</a> for details."
```